pkg/apis/clusteroperator/types.go

```diff
@@ -121,7 +121,17 @@ type VMImages struct {
 
 // AWSVMImages indicates which AMI to use in each supported AWS region for this OpenShift version.
 type AWSVMImages struct {
-	AMIByRegion map[string]string
+	RegionAMIs []AWSRegionAMIs
+}
+
+// AWSRegionAMIs defines which AMI to use for a node type in a given region.
+type AWSRegionAMIs struct {
+	Region string
+	// ComputeAMI is the ID of the AMI to use for compute nodes in the cluster. If no MasterAMI is defined, it will be used for masters as well.
+	ComputeAMI string
```

Comment:
What are your thoughts on calling this AMI instead? I think that since it is the default AMI for all nodes, it makes more sense to call it AMI.